### PR TITLE
Story/19843

### PR DESCRIPTION
**Pull request by cleitondutra-amcom**
[AB#19843](https://dev.azure.com/amcomgov/c1dcf343-60ee-40b6-a30a-3b9c12c6d220/_workitems/edit/19843)

### CODE REVIEW

File: src/SME.SGP.Aplicacao/Queries/CartaIntencoesObservacao/ListarCartaIntencoesObservacao/ListarCartaIntencoesObservacaoQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+        public ListarObservacaoCartaIntencoesQueryHandler(IRepositorioCartaIntencoesObservacao repositorioCartaIntencoesObservacao)
+        {
+            this.repositorioCartaIntencoesObservacao = repositorioCartaIntencoesObservacao ?? throw new ArgumentNullException(nameof(repositorioCartaIntencoesObservacao));
+        }
+
+        //public async Task<IEnumerable<ListarObservacaoCartaIntencoesDto>> Handle(ListarObservacaoCartaIntencoesQuery request, CancellationToken cancellationToken)
+        //{
+        //    return await repositorioCartaIntencoesObservacao.ListarPorCartaIntencoesAsync(request.CartaIntencoesId, request.UsuarioLogadoId);
+        //}
```
Comment (nogueiraamcom) [inline src/SME.SGP.Aplicacao/Queries/CartaIntencoesObservacao/ListarCartaIntencoesObservacao/ListarCartaIntencoesObservacaoQueryHandler.cs:24]: Remover código comentado

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            }
+        },
+        {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        }
+    ]
+}
```
Comment (nogueiraamcom) [inline .vscode/launch.json:36]: Remover arquivo de configuração do vscode e adicionar no gitignore

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                "watch",
+                "run",
+                "${workspaceFolder}/src/SME.SGP.Api/SME.SGP.Api.csproj",
+                "/property:GenerateFullPaths=true",
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "problemMatcher": "$msCompile"
+        }
+    ]
+}
```
Comment (nogueiraamcom) [inline .vscode/tasks.json:42]: Remover arquivo de configuração do vscode e adicionar no gitignore

File: scripts/V226__OBSERVACOES_CARTA_INTENCOES.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	CONSTRAINT carta_intencoes_observacao_pk PRIMARY KEY (id)
+);
+
+
+select
+	f_cria_fk_se_nao_existir(
+		'carta_intencoes_observacao',
+		'carta_intencoes_observacao_turma_fk',
+		'FOREIGN KEY (turma_id) REFERENCES turma (id)'
+	);
```
Comment (nogueiraamcom) [inline scripts/V226__OBSERVACOES_CARTA_INTENCOES.sql:23]: Adicionar indices para as colunas turma_id e componente_curricular_id

File: .vscode/launch.json (hunk 2 above)
Comment (luizaraujoamcom) [reply .vscode/launch.json]: https://github.com/prefeiturasp/SME-NovoSGP/pull/3723/commits/68ddd24b2b3aecf26f72ea32ce32c3c9d8e01f98

File: .vscode/tasks.json (hunk 3 above)
Comment (luizaraujoamcom) [reply .vscode/tasks.json]: https://github.com/prefeiturasp/SME-NovoSGP/pull/3723/commits/68ddd24b2b3aecf26f72ea32ce32c3c9d8e01f98

File: scripts/V226__OBSERVACOES_CARTA_INTENCOES.sql (hunk 4 above)
Comment (luizaraujoamcom) [reply scripts/V226__OBSERVACOES_CARTA_INTENCOES.sql]: https://github.com/prefeiturasp/SME-NovoSGP/pull/3723/commits/233c8e29e610e814e1b5462817c3c64568a283b7

File: src/SME.SGP.Aplicacao/Queries/CartaIntencoesObservacao/ListarCartaIntencoesObservacao/ListarCartaIntencoesObservacaoQueryHandler.cs (hunk 1 above)
Comment (luizaraujoamcom) [reply src/SME.SGP.Aplicacao/Queries/CartaIntencoesObservacao/ListarCartaIntencoesObservacao/ListarCartaIntencoesObservacaoQueryHandler.cs]: https://github.com/prefeiturasp/SME-NovoSGP/pull/3723/commits/68ddd24b2b3aecf26f72ea32ce32c3c9d8e01f98

**Pull request merged**